src/main/java/CellContent.java

```diff
@@ -0,0 +1,97 @@
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellType;
+
+public class CellContent {
+	public CellType type;
```

Comment:
I'm personally not a proponent of using getters and setters for very simple data constructs. In C/C++ this would be a struct with a Union. The data structure was introduced for convenience, if I'm reintroducing unecessary complexity I might as well use the Cell class directly. But I'm aware that people that take OOP very seriously would be very upset with me. In the end I personally think trusting the programmers competence and making it as easy to use as possible is more important than shielding against improper use.